src/core/reference/src/runtime/reference/convert.cpp

```diff
@@ -5,17 +5,18 @@
 #include "ngraph/runtime/reference/convert.hpp"
 
 #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
-
 #    include "jit_generator.hpp"
+#endif
 
 namespace ngraph {
 namespace runtime {
 namespace reference {
+#if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
```

Comment:
Just curious how it was built before, we haven't added any specifically new x86-dependent stuff.

Reply:
Previously this line was at the beginning and the whole file was inside this macros - which means this file indeed was compiled only for x86 archs.

I moved it here after namespaces so that i don't have open and close namespaces again for ARM code.